lldb/source/Plugins/ObjectFile/wasm/ObjectFileWasm.cpp

```diff
@@ -304,7 +313,7 @@ ParseData(SectionSP data_section_sp) {
     if (flags > std::numeric_limits<uint32_t>::max())
       return llvm::createStringError("segment size overflows uint32_t");
```

Comment:
This error should say flags, right?

Reply:
Actually, no, the error message is correct, but the check above is not (it should check `segment_size`).